easy-ui-react/src/Menu/Menu.stories.tsx

```diff
@@ -230,3 +231,58 @@ export const CustomPlacement: StoryObj<MenuOverlayProps<unknown>> = {
   },
   decorators: [OverlayLayoutDecorator],
 };
+
+export const MultipleSelection: Story = {
+  render: () => {
+    const [selectedKeys, setSelectedKeys] = React.useState<Selection>(
+      new Set([]),
+    );
+
+    return (
+      <Menu>
+        <Menu.Trigger>
+          <DropdownButton>Click me</DropdownButton>
+        </Menu.Trigger>
+        <Menu.Overlay
+          onSelectionChange={setSelectedKeys}
+          selectionMode="multiple"
+          selectedKeys={selectedKeys}
```

Comment:
could you add an action handler to these multiselection stories so we can see the selection result in the actions panel? i.e. `onAction={action("Selected")}`